submarine-dist/src/assembly/distribution.xml

```diff
@@ -135,6 +135,20 @@
         <include>python-interpreter-${project.version}-shade.jar</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>../submarine-commons/cluster-server/target</directory>
+      <outputDirectory>/workbench/lib</outputDirectory>
+      <includes>
+        <include>cluster-server-${project.version}-shade.jar</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <directory>../submarine-commons/cluster-server/target</directory>
+      <outputDirectory>/lib</outputDirectory>
```

Comment:
How about we change /lib to commons-lib. And all the jar from submarine-commons will be put in it.

Reply:
Changed to `/lib/commons`.